install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -0,0 +1,107 @@
+            name: strimzi-cluster-operator
+      containers:
+        - name: strimzi-cluster-operator
+          image: registry.redhat.io/amq7/amq-streams-rhel7-operator:1.6.0
+          ports:
+            - containerPort: 8080
+              name: http
+          args:
+            - /opt/strimzi/bin/cluster_operator_run.sh
+          volumeMounts:
+            - name: co-config-volume
+              mountPath: /opt/strimzi/custom-config/
+          env:
+            - name: STRIMZI_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: STRIMZI_FULL_RECONCILIATION_INTERVAL_MS
+              value: "120000"
+            - name: STRIMZI_OPERATION_TIMEOUT_MS
+              value: "300000"
+            - name: STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE
+              value: strimzi/kafka:0.20.0-kafka-2.6.0
+            - name: STRIMZI_DEFAULT_KAFKA_EXPORTER_IMAGE
+              value: strimzi/kafka:0.20.0-kafka-2.6.0
+            - name: STRIMZI_DEFAULT_CRUISE_CONTROL_IMAGE
+              value: strimzi/kafka:0.20.0-kafka-2.6.0
+            - name: STRIMZI_DEFAULT_TLS_SIDECAR_CRUISE_CONTROL_IMAGE
+              value: strimzi/kafka:0.20.0-kafka-2.6.0
+            - name: STRIMZI_KAFKA_IMAGES
+              value: |
+                2.5.0=strimzi/kafka:0.20.0-kafka-2.5.0
+                2.5.1=strimzi/kafka:0.20.0-kafka-2.5.1
+                2.6.0=strimzi/kafka:0.20.0-kafka-2.6.0
+            - name: STRIMZI_KAFKA_CONNECT_IMAGES
+              value: |
+                2.5.0=strimzi/kafka:0.20.0-kafka-2.5.0
+                2.5.1=strimzi/kafka:0.20.0-kafka-2.5.1
+                2.6.0=strimzi/kafka:0.20.0-kafka-2.6.0
+            - name: STRIMZI_KAFKA_CONNECT_S2I_IMAGES
+              value: |
+                2.5.0=strimzi/kafka:0.20.0-kafka-2.5.0
+                2.5.1=strimzi/kafka:0.20.0-kafka-2.5.1
+                2.6.0=strimzi/kafka:0.20.0-kafka-2.6.0
+            - name: STRIMZI_KAFKA_MIRROR_MAKER_IMAGES
+              value: |
+                2.5.0=strimzi/kafka:0.20.0-kafka-2.5.0
+                2.5.1=strimzi/kafka:0.20.0-kafka-2.5.1
+                2.6.0=strimzi/kafka:0.20.0-kafka-2.6.0
+            - name: STRIMZI_KAFKA_MIRROR_MAKER_2_IMAGES
+              value: |
+                2.5.0=strimzi/kafka:0.20.0-kafka-2.5.0
+                2.5.1=strimzi/kafka:0.20.0-kafka-2.5.1
+                2.6.0=strimzi/kafka:0.20.0-kafka-2.6.0
+            - name: STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE
+              value: strimzi/operator:0.20.0
+            - name: STRIMZI_DEFAULT_USER_OPERATOR_IMAGE
+              value: strimzi/operator:0.20.0
+            - name: STRIMZI_DEFAULT_KAFKA_INIT_IMAGE
+              value: strimzi/operator:0.20.0
+            - name: STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE
+              value: strimzi/kafka-bridge:0.19.0
+            - name: STRIMZI_DEFAULT_JMXTRANS_IMAGE
+              value: strimzi/jmxtrans:0.20.0
```

Comment:
You will change these later?

Reply:
nope they have to be changed! damn! that was a new file :-)